HISTORY.md

```diff
@@ -0,0 +1,90 @@
+## Release History
+
+### Version next
+
+### Version 2.1.0
```

Comment:
What happened to create this duplicate version?

Reply:
Looks like this commit by @TheConnMan was merged to dev and hadn't yet made it to master:
https://github.com/netuitive-community-packages/netuitive-packages-windows/pull/32/commits/57dd89471040f4e263c25d0ae3262be4cfc1b81d

Strangely, the changes made by the above commit don't currently exist on dev and it's unclear to me why that is, but probably better solved by action not taken in this PR. Went ahead and made the history match what was in the readme, then added a version next.